static/sass/_pattern_takeovers.scss

```diff
@@ -3,13 +3,15 @@
 @mixin p-takeovers($name) {
   .p-takeover--#{$name} {
     @extend %vf-strip;
+
     background-blend-mode: multiply, multiply, normal, normal;
-    @content;
-    background-position: top right, top left, right bottom -1px, left top;
+
+    @content background-position: top right, top left, right bottom -1px, left top;
```

Comment:
`@content` needs to be on it's own line with a semi-colon, does it not?

Reply:
Done